README.md

```diff
@@ -214,49 +122,49 @@ All the javascript dependencies are listed in `package.json`. To install them ru
 
 
 
-##### Set up the database and start redis
+### 5. Set up the database and start redis
 
-  1. Install [postgres](https://www.postgresql.org/download/) if you don't have
-     it already. If you're using a package manager, you need to make sure you install
-     the following packages: `postgresql`, `postgresql-contrib`, and `postgresql-server-dev-all`
-     which will be required to build `psycopg2` python driver.
+Install [postgres](https://www.postgresql.org/download/) if you don't have it already. If you're using a package manager, you need to make sure you install the following packages: `postgresql`, `postgresql-contrib`, and `postgresql-server-dev-all` which will be required to build `psycopg2` python driver.
 
-  2. Make sure postgres is running
+Make sure postgres is running
 
-         service postgresql start
-         # or pg_ctl -D /usr/local/var/postgresql@9.6 start
+    service postgresql start
+    # or pg_ctl -D /usr/local/var/postgresql@9.6 start
 
-  3. Create a database user with username `learningequality` and password `kolibri`:
+Create a database user with username `learningequality` and password `kolibri`:
 
-         sudo su postgres
-         psql
-         # mac: psql postgres
-           CREATE USER learningequality with NOSUPERUSER INHERIT NOCREATEROLE CREATEDB LOGIN NOREPLICATION NOBYPASSRLS PASSWORD 'kolibri';
+    sudo su postgres
+    psql
+    # mac: psql postgres
+      CREATE USER learningequality with NOSUPERUSER INHERIT NOCREATEROLE CREATEDB LOGIN NOREPLICATION NOBYPASSRLS PASSWORD 'kolibri';
 
-  4. Create a database called `gonano`
+Create a database called `gonano`
 
-         CREATE DATABASE "gonano" WITH TEMPLATE = template0 OWNER = "learningequality";
+    CREATE DATABASE "gonano" WITH TEMPLATE = template0 OWNER = "learningequality";
 
 
 
-##### Run all database migrations and load constants
+### 6. Run all database migrations and load constants
 
-You'll only need to run these commands once, to setup the necessary tables and
-constants in the database:
+You'll only need to run these commands once, to setup the necessary tables and constants in the database:
 
     # On one terminal, run all external services
     $ yarn run services
 
     # On another terminal, run devsetup to create all the necessary tables and buckets
     $ yarn run devsetup
 
-##### Start the dev server
+
+
+### 7. Start the dev server
 
 You're all setup now, and ready to start the Studio local development server:
 
     make devserver
 
+Alternatively, you can run `pipenv run make devserver` or `yarn make devserver`
```

Comment:
I think we should standardize on one command. This gives the reader three ways to do it but no guidance on which one to choose or the pros and cons.

Reply:
BTW, it's `yarn run devserver` and I think we should poll people to see what they use. I think `yarn run devserver` is the typical approach, and it matches with Kolibri so that is helpful too.

Reply:
I remember there being a reason for running `pipenv run make devserver`, but I can't remember what it was